operations/configuration.md

```diff
@@ -429,13 +476,14 @@ Definition of single tax rate.
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `Discriminator` | string [Tax rate strategy discriminator](#tax-rate-strategy-discriminator) | required | If tax rate is flat or relative. |
+| `Discriminator` | string [Tax rate strategy discriminator](#tax-rate-strategy-discriminator) | required | If tax rate is flat, relative or dependent. |
 | `Value` | object | required | Structure of the object depends on [Tax rate strategy discriminator](#tax-rate-strategy-discriminator). |
 
 #### Tax rate strategy discriminator
 
 * `Flat` - Used with [Flat tax rate strategy data](#flat-tax-rate-strategy-data) (e.g. 5.00 EUR). 
-* `Relative` - Used with [Relative tax rate strategy data](#relative-tax-rate-strategy-data) (e.g. 21%).
+* `Relative` - Used with [Relative tax rate strategy data](#relative-tax-rate-strategy-data) (e.g. 21%). Relative tax is calculated only from net base value.
+* `Dependent` - Used with [Dependent tax rate strategy data](#dependent-tax-rate-strategy-data) (e.g. 10%). Dependent tax is calculated from net base value and all taxes it depends on.
```

Comment:
I was thinking about not creating another discriminator to API and just adding some BaseTaxationCodes to the Relative one. Because in the end the Dependent tax rate is also relative. But I guess it's easier to understand like this, right? @marektresnak @starychfojtu @siroky

Reply:
I agree it's easier to understand as a separate strategy rather than saying that all of our relative taxes are dependent, but the vast majority doesn't depend on anything.